mailpoet/lib/Segments/SegmentsRepository.php

```diff
@@ -54,7 +61,7 @@ protected function getEntityClassName() {
    * @return SegmentEntity[]
    */
   public function findByTypeNotIn(array $types): array {
-    return $this->doctrineRepository->createQueryBuilder('s')
+      return $this->doctrineRepository->createQueryBuilder('s')
```

Comment:
I guess we need to fix the indentation here. No idea why PHPCS didn't catch this. Maybe because of our custom ruleset. 

```suggestion
    return $this->doctrineRepository->createQueryBuilder('s')
```

Reply:
Fixed in a7fed3f26d5252f6f5cd19762f8169c7aee87824